Files/ViewModels/SettingsViewModel.cs

```diff
@@ -685,6 +685,15 @@ public bool HideConfirmElevateDialog
             set => Set(value);
         }
 
+        /// <summary>
+        /// Gets or sets a value indicating the default volume on media preview.
+        /// </summary>
+        public double MediaVolume
+        {
+            get => Math.Min(Math.Max(Get(1.0d), 0.0d), 1.0d);
```

Comment:
why not just?
```
get => Get(1.0);
```

Reply:
to avoid any bugs or issues regarding the media player increasing volume to 200%.
just to be safe and not to destroy other peoples hearing

Reply:
I think it won't go above 1.0d even if you set the value to like 200.0d. You could try it and compare both cases

Reply:
player/app crashes if set higher then 1.0